dacsspace/validator.py

```diff
@@ -1,8 +1,28 @@
 
+from jsonschema import ValidationError, validate
+
 
 class Validator:
     """Validates data from ArchivesSpace."""
+# Creates a simple schema to validate json against
+
+    def get_schema(self):
+        schema = {
+            "type": "object",
+            "title": "DACSspace schema",
+            "required": [
+                "title",
+                "id_0"
+            ],
+            "properties": {
+                "title": {
+                    "type": "string"
+                }
+            }
+        }
+        return schema
 
+# Checks data against schema
```

Comment:
Same with this comment - if it's necessary for someone to understand the code then we should move it to the docstring. If not, it should be removed.